README.md

```diff
@@ -1 +1,80 @@
-# strong-params-basics
+# Strong Params Basics
+
+## What are Strong Params?
+
+To understand the goal of strong params, let's pretend that you run a pharmacy. What would happen if you let all prescription orders come through without checking for: valid prescriptions, driver licenses, etc.? (Spoiler alert: you'd probably end up in jail). It would be insane to run a pharmacy without verifying that orders were legitimate. In the same way Rails application (starting in Rails 4+) wanted to shore up some security vulnerabilities and now require that you whitelist the parameters that are permitted when you're sending form data to the database.
+
+
+## Setup
+
+To prevent confusion, in previous lessons I manually turned off the strong parameter requirement, to start this lesson I've gone with the Rails 4+ default that requires you to use strong parameters. If for any reason you ever need to turn them off you can add the line `config.action_controller.permit_all_parameters = true` to the `config/application.rb` file.
+
+
+## Code Implementation
+
+Let's begin by running the rails server and navigating to `localhost:3000/posts/new`, once there fill out the form and click `submit`. You'll see we get the following `ForbiddenAttributesError`:
```

Comment:
As a way to show the power of strong params, want to include an example in here with strong params turned off...then show them what happens if you open inspect element (they know what that is) add a "rogue field" and then hit submit. Oh man! it now updates the `user_id` or w/e Then turn on strong params and show them it's forbidden. That's what strong params does